### PR TITLE
ENH: Add ophyd build for latest release candidate

### PR DESCRIPTION
**Pull request by teddyrendahl**

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]

File: ophyd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% set version = "1.4.0rc1" %}
+package:
+  name: ophyd 
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluesky/ophyd/archive/v{{ version }}.tar.gz
+  fn: pydm-{{ version }}.tar.gz
```
Comment (klauer) [inline ophyd/meta.yaml]:
```suggestion
  fn: ophyd-{{ version }}.tar.gz
```

**Review by klauer — CHANGES_REQUESTED** [review]

File: ophyd/meta.yaml (hunk 1 above)
Comment (ZLLentz) [reply ophyd/meta.yaml]: Wow, I'm blind. Retracting approval.

**Review by ZLLentz — CHANGES_REQUESTED** [review]
down with pydm

**Comment by teddyrendahl** [comment]
And I would have gotten away with it too, if it wasn't for you and your lousy code reviews!

**Review by ZLLentz — APPROVED** [review]

**Comment by teddyrendahl** [comment]
@klauer Think we need your approval here to merge

**Review by klauer — APPROVED** [review]
Oh... ok!

**Pull request merged**